java/lang/missing_database_authentication.yml

```diff
@@ -0,0 +1,30 @@
+patterns:
+  - pattern: |
+      $<SQL_DRIVER_MANAGER>.getConnection($<_>, $<_>, $<EMPTY_STRING>)
+    filters:
+      - variable: SQL_DRIVER_MANAGER
+        detection: java_lang_missing_database_authentication_sql_driver_manager
+      - variable: EMPTY_STRING
+        regex: ('|")('|")
+languages:
+  - java
+severity: warning
+auxiliary:
+  - id: java_lang_missing_database_authentication_sql_driver_manager
+    patterns:
+      - DriverManager
```

Comment:
What's the reason for making this a separate rule?

Reply:
Ah, whoops - I had regex there but took it out. I'll update it!